### ISSUE TITLE
Back-end revisions to index

### ISSUE THREAD

**Issue by bnedavies** [Gizra/IBLI#258]
@niryariv We discussed having access at the back end to adjust the index.  Where will this be on the webplatform?  admin use only

**Comment by niryariv** [comment]
do you mean manually changing index values, or the algorithm calculating the index? I don't remember discussing this, shouldn't be hard to implement (at least the index values scenario).

**Comment by bnedavies** [comment]
yes.  @apurbashee can you talk to Andrew about what exactly he envisions here and relay that to Nir?

**Comment by bnedavies** [comment]
please assign to apurba

**Comment by niryariv** [comment]
sorry, mis-clicked :)
